src/modules/extra/m_ssl_openssl.cpp

```diff
@@ -336,7 +336,7 @@ namespace OpenSSL
 		/** OpenSSL makes us have two contexts, one for servers and one for clients
 		 */
 		Context ctx;
-		Context clictx;
+		Context clientctx;
```

Comment:
both spellings were used.

Note that I'm not picky / do tend to favor the spelling with more uses. Sometimes this means longer, sometimes shorter. Dropping isn't a big deal.